src/legacy/core_plugins/vis_type_table/public/legacy_imports.ts

```diff
@@ -19,13 +19,10 @@
 
 export { npSetup, npStart } from 'ui/new_platform';
 export { getFormat } from 'ui/visualize/loader/pipeline_helpers/utilities';
-export { IAggConfig, AggGroupNames, Schemas } from 'ui/agg_types';
 // @ts-ignore
 export { PaginateDirectiveProvider } from 'ui/directives/paginate';
 // @ts-ignore
 export { PaginateControlsDirectiveProvider } from 'ui/directives/paginate';
```

Comment:
you could also adapt imports `PaginateDirectiveProvider` and `PaginateControlsDirectiveProvider` from `kibana_legacy`. then `ui/directives/paginate` is only used in `src/legacy/core_plugins/timelion/public/directives/saved_object_finder.js`, so it could be inlined there

Reply:
It's a really good catch, removed reference here and completely removed `ui/directives/paginate` references.